fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
Oh, "fcr:metadata" is purely an implementation convenience. You have to generate a URI for the first, required, description, and we do it by adding that string of characters. We don't want to privilege it in any way. We could do it (e.g.) by using a UUID, and sometimes I think we would do better to do something that opaque and thereby to force people to use the header as proper RESTafarians do. @mikedurbin is pointing to a deeper question (is a description only _related_ to its descriptand, or is it actually a _representation_ of it, or is it both?) that we probably want to try to drive around for this PR.